bentoml/server/bento_api_server.py

```diff
@@ -195,21 +194,54 @@ def bento_service_api_wrapper(api, service_name, service_version):
     summary_name = str(service_name) + "_" + str(api.name)
     request_metric_time = Summary(summary_name, summary_name + " request latency")
 
+    def log_image(request, request_id):
+        img_prefix = 'image/'
+        log_folder = config['logging'].get('base_log_folder')
```

Comment:
another change after https://github.com/bentoml/BentoML/pull/287 is to use config('logging').get('base_log_folder')

Reply:
Ok, I'll rebase and update.